test/vault/integration.clj

```diff
@@ -0,0 +1,150 @@
+(ns vault.integration
+  "Integration test support code. Manages running a local Vault server in
+  development mode in order to truly exercise the client code."
+  (:require
+    [clojure.java.io :as io]
+    [clojure.java.shell :as shell]
+    [clojure.string :as str]
+    [vault.client.http]
+    [vault.core :as vault])
+  (:import
+    (java.net
+      InetSocketAddress
+      Socket
+      SocketTimeoutException)
+    java.util.List
+    java.util.concurrent.TimeUnit))
+
+
+;; ## Development Server
+
+(def interface
+  "Local interface to bind the server to."
+  "127.0.0.1")
+
+
+(def port
+  "Local port to bind the server to."
+  8201)
+
+
+(def address
+  "Local address the development server is bound to."
+  (str "http://" interface ":" port))
+
+
+(def root-token
+  "Root token set for the development server."
+  "t0p-53cr3t")
+
+
+(defn start-server!
+  "Start a local Vault development server process. Returns the child process
+  object."
+  ^Process
+  []
+  (let [command ["vault" "server" "-dev"
+                 (str "-dev-listen-address=" (str interface ":" port))
+                 (str "-dev-root-token-id=" root-token)
+                 "-dev-no-store-token"]
+        work-dir (io/file "target/vault")
```

Comment:
looks like the integration is failing since this directory isn't created (at or around) https://github.com/amperity/vault-clj/pull/57/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R71

Reply:
It was because the `vault` CLI wasn't on the `PATH` - fixed in the latest configs.